sympy/core/power.py

```diff
@@ -193,6 +195,9 @@ def __new__(cls, b, e, evaluate=None):
         b = _sympify(b)
         e = _sympify(e)
         if evaluate:
+            from sympy import zoo
+            if e is zoo:
```

Comment:
We can avoid importing `zoo` by using `S.ComplexInfinity`.

Reply:
Yes.